index.html

```diff
@@ -53,6 +53,7 @@ <h4>Nearby</h4>
       <h2>
         <a name="status" id="status">Status</a>
       </h2>
+      <p>October 2023: Document updated to account for <a href="https://www.w3.org/2023/Process-20230612/#changes-2021">changes in Process 2023</a>, in particular the Director-free changes.</p>
```

Comment:
```suggestion
      <p>October 2023: Document updated to account for <a href="https://www.w3.org/Consortium/Process/#changes-2021">changes in Process 2023</a>, in particular the Director-free changes.</p>
```

Reply:
Since that one mentions changes in a particular dated version, it seems the dated link is appropriate there?